NXDOC-2707: release notes for 3.0.35 (#2275)
nuxeo/doc.nuxeo.com-content: src/nxdoc/web-ui/web-ui-release-notes/web-ui-release-notes-3-0-35.md

---
title: Version 3.0.35
description: Discover what's new in Web UI 3.0.35.
review:
  comment: ''
  date: '2024-07-25'
  status: ok
toc: true
labels:
tree_item_index: 970
hidden: true
---

{{{multiexcerpt 'matching-notes' page='web-ui-release-notes'}}}

{{! multiexcerpt name='web-ui-updates'}}

## What’s New in Web UI for LTS 2021 (Version 3.0.35)

> - The + button for document creation will be displayed when using mobile browsers.<br/>[[WEBUI-1489](https://jira.nuxeo.com/browse/WEBUI-1489)]

Reviewer: ```suggestion
- The document creation button is always displayed when using mobile browsers.<br/>[[WEBUI-1489](https://jira.nuxeo.com/browse/WEBUI-1489)]
```

### Other Noteworthy Changes

> - Fulltext input will be applied and search will be returning corresponding results on clicking search button after disabling auto search on nuxeo-default-search-form.<br/>[[WEBUI-1490](https://jira.nuxeo.com/browse/WEBUI-1490)]

Reviewer: ```suggestion
- Search is launched with the current fulltext input after disabling auto search on the default search form.<br/>[[WEBUI-1490](https://jira.nuxeo.com/browse/WEBUI-1490)]
```
